src/crud_sql.py

```diff
@@ -0,0 +1,353 @@
+import re
+
+from fastapi import HTTPException
+from sqlalchemy.orm import Session
+
+from src import schemas
+
+EMAIL_REGEX = re.compile(r"[^@]+@[^@]+\.[^@]+")
+
+
+def schema_processor_for_inserting(schema):
+    # schema example structure: [("Name", "Pepperoni"), ("Price", "23.0")]
+    headers, values = [], []
+    for value in schema:
+        if value[1] is None or value[0] == "OrderedItems":
+            continue
```

Comment:
please unpack the array to improve readability. `value[1]` and `value[0]` doesn't tell much 
```a,b = [1,2]```